.paket/Paket.Restore.targets

```diff
@@ -289,16 +289,14 @@
     <PropertyGroup>
       <PaketProjectFile>$(MSBuildProjectDirectory)/$(MSBuildProjectFile)</PaketProjectFile>
       <ContinuePackingAfterGeneratingNuspec>true</ContinuePackingAfterGeneratingNuspec>
-      <UseMSBuild16_10_Pack>false</UseMSBuild16_10_Pack>

```

Comment:
Looks like the old Paket.Restore.targets got committed and so we lost the changes from 6.0.0-rc2. Can you revert this?

Reply:
Whoops! Sorry... this should be corrected now.